StRoot/StFstRawHitMaker/StFstRawHitMaker.cxx

```diff
@@ -0,0 +1,598 @@
+		mMinNumOfRawHits      = fstControlTable[0].kFstMinNumOfRawHits;
+		mMaxNumOfRawHits      = fstControlTable[0].kFstMaxNumOfRawHits;
+	}
+
+	// FST pedestal/rms table
+	const fstPedNoise_st *gPN = mFstDb->getPedNoise();
+
+	if ( !gPN ) {
+		LOG_ERROR << "Pointer to FST pedestal/noise table is null" << endm;
+		ierr = kStErr;
+	}
+	else {
+		for (int i = 0; i < kFstNumApvs; i++) {
+                     for ( int j = 0; j < kFstNumRStripsPerSensor; j++) {
+                          for ( int k = 0; k < kFstNumTimeBins; k++) {
+   			       LOG_DEBUG << Form(" Print entry %d-%d-%d : CM noise=%f ", i, j, k, (float)gPN[0].cmNoise[(i*kFstNumRStripsPerSensor+j)*kFstNumTimeBins+k] / 100.) << endm;
+			       mCmnVec[i][j][k] = (float)gPN[0].cmNoise[(i*kFstNumRStripsPerSensor+j)*kFstNumTimeBins+k] / 100.0;
+                          }
+                     }
+		}
+
+		for (int i = 0; i < kFstNumElecIds; i++) {
+                     for ( int j = 0; j < kFstNumTimeBins; j++) {
+			LOG_DEBUG << Form(" Print entry %d-%d : pedestal=%f ", i, j, (float)gPN[0].pedestal[i*kFstNumTimeBins+j]) << endm;
+			mPedVec[i][j] = (float)gPN[0].pedestal[i*kFstNumTimeBins+j];
+                     }
+		}
+
+		for (int i = 0; i < kFstNumElecIds; i++) {
+                     for ( int j = 0; j < kFstNumTimeBins; j++) {
+			LOG_DEBUG << Form(" Print entry %d-%d : RMS noise=%f ", i, j, (float)gPN[0].totNoise[i*kFstNumTimeBins+j] / 100.) << endm;
+			mTotRmsVec[i][j] = (float)gPN[0].totNoise[i*kFstNumTimeBins+j] / 100.;
+                     }
+		}
+                for (int i = 0; i < kFstNumElecIds; i++) {
+                     for ( int j = 0; j < kFstNumTimeBins; j++) {
+                        LOG_DEBUG << Form(" Print entry %d-%d : RMS noise=%f ", i, j, (float)gPN[0].ranNoise[i*kFstNumTimeBins+j] / 100.) << endm;
+                        mRanRmsVec[i][j] = (float)gPN[0].ranNoise[i*kFstNumTimeBins+j] / 100.;
+                     }
+                }
+	}
+
+	// FST gain table
+	const fstGain_st *gG = mFstDb->getGain();
+
+	if ( !gG ) {
+		LOG_WARN << "Pointer to FST gain table is null" << endm;
+		ierr = kStWarn;
+	}
+	else {
+		for (int i = 0; i < kFstNumElecIds; i++) {
+			LOG_DEBUG << Form(" Print entry %d : gain=%f ", i, (float)gG[0].gain[i]) << endm;
+			mGainVec[i] = (float)gG[0].gain[i];
+		}
+	}
+
+	// FST mapping table
+	const fstMapping_st *gM = mFstDb->getMapping();
+
+	if ( !gM ) {
+		LOG_ERROR << "Pointer to FST mapping table is null" << endm;
+		ierr = kStErr;
+	}
+	else {
+		for (int i = 0; i < kFstNumElecIds; i++) {
+			LOG_DEBUG << Form(" Print entry %d : geoId=%d ", i, gM[0].mapping[i]) << endm;
+			mMappingVec[i] = gM[0].mapping[i];
+		}
+	}
+
+	// FST chip configuration status table
+	const fstChipConfig_st *gCS = mFstDb->getChipStatus();
+
+	if ( !gCS ) {
+		LOG_ERROR << "Pointer to FST chip configuration table is null" << endm;
+		ierr = kStErr;
+	}
+	else {
+		//if (runnumber == gCS[0].run) {
```

Comment:
Of no needed, please just remove this line.

Reply:
done in a1b656422171901b3c0f9af39228875d9fe1be5e